src/main/java/at/hannibal2/skyhanni/config/features/garden/visitor/ShoppingListConfig.java

```diff
@@ -46,6 +46,12 @@ public class ShoppingListConfig {
     @ConfigEditorBoolean
     public boolean showSackCount = true;
 
+    @Expose
+    @ConfigOption(name = "Show Super Craft", desc = "Show super craft button if there are enough materials to make in the sack.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean showSuperCraft = true;
```

Comment:
I'd argue to have this setting default disabled. or at least also check if the exact type and amount of neeed items is avaliable already and hide this button in that case